.buildkite/pipeline.yml

```
@@ -2,7 +2,11 @@
 common_params:
   # Common plugin settings to use with the `plugins` key.
   - &common_plugins
-    - automattic/bash-cache#2.3.0
+    - automattic/bash-cache#add/buildkite-test-analytics
+  # Common artifact paths used across steps
+  - &artifact_paths
+    - build/test-results/**/*
+    - build/reports/**/*
```

```suggestion
    - **/build/test-results/**/*
    - **/build/reports/**/*
```

@jkmassel I don't think the current implementation works because the artifacts were not uploaded in [this build](https://buildkite.com/automattic/wordpress-fluxc-android/builds/1512#01823cc9-641d-443a-8c3e-1a895f8fc2a6). I've been using `**/build/reports/lint-results*.*` to upload lint results in other projects, so I think adding the `**/` as a prefix would fix this.